pom.xml

```diff
@@ -99,13 +99,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>3.4.0</version>
-        </dependency>
-
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>scm-api</artifactId>
-            <version>2.2.0</version>
+            <version>4.0.0-rc</version>
```

Comment:
Too new dependency, github plugin wouldn't lock users to have release candidate as hard dependency.